### DISCUSSION TITLE
[4.10] certificateCreate form setup

### DISCUSSION THREAD

**Discussion by komalm** [Sunbird-Ed/Community#573] [Sunbird ED configuration discussion] [answered]
Hello @vinukumar-vs,

**Site Environments Details** - SunbirdEd portal of version 4.10

**Form Setup -**

Can you please help us to setup this following form.

Payload - 

```
request: 
{type: "certificate", action: "certificateCreate", subType: "course", rootOrgId: "*",…}
action: "certificateCreate"
component:"portal"
rootOrgId:"*"
subType:"course"
type:"certificate"
```

**Comment by rajeevsathish** [comment, marked as answer]
@komalm  Please use the form create with this request below
`{
    "request":{
        "type": "certificate",
        "action": "certificateCreate",
        "subType": "course",
        "rootOrgId": "*",
        "component": "portal",
        "data": {
                "enableSVGEditor": false
            }
    }
}`
CC: @vinukumar-vs @Krishnaj20 @chitranshu-keshav
